Automattic/vip-go-mu-plugins
# Add/admin notice conditions

## Description

Adds support for `wp-version` and `capabaility` restrictions for admin notices.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR
1. Add admin notice in admin-notice/admin-notice.php
```
$admin_notice_controller->add(
	new Admin_Notice(
		'Test',
		[
			... add conditions here
		]
) );
```
1. Add conditions
1. See the notice is shown if all conditions are true doesn show otherwise.
